MvvmCross-Forms/MvvmCross.Forms/ViewModels/IMvxMasterDetailViewModel.cs

```diff
@@ -1,26 +0,0 @@
-﻿using System;
-using MvvmCross.Core.ViewModels;
-
-namespace MvvmCross.Forms.ViewModels
-{
-    /// <summary>
-    /// Extends IMvxViewModel interface to have track of the ViewModel used in Detail as root.
-    /// Also, it exposes a method to be  launched when this root page is activated due to navigation popping in Detail
-    /// 
-    /// </summary>
-    internal interface IMvxMasterDetailViewModel : IMvxViewModel
```

Comment:
Isn't this a breaking change? Same with `MvxMasterDetailViewModel`

Reply:
This already was removed in 5.3 but i forgot to remove the viewmodels.